src/views/ops/Job/JobUpdateCreate.vue

```diff
@@ -99,6 +99,7 @@ export default {
                 data?.variable.map(item => {
                   delete item.job
                   delete item.playbook
+                  delete item.id
                   return item
                 })
                 updateForm({ variable: data.variable })
```

Comment:
The updated line between lines 98-102 is:

```
delete item.job
```

This will likely cause an error due to deleting elements from data that were not passed in through `data`. You would need either to ensure each element of `item` gets passed in separately (`{ variable: item }`) or to remove `job`, if there's only one field you want it gone.

Additionally, instead of using `{ variable: data.variable }` directly within your updateForm method, use arrow syntax and chain functions to access the properties like this:

```javascript
updateForms(data.form, { variables: Object.entries(data.variables) })
```

If all fields exist with these two updates, then there should be no unexpected issues.